experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -140,6 +140,8 @@ export const FormAudience = ({
     fieldWarnings,
   );
 
+  const isMobile = window.innerWidth <= 768;
```

Comment:
and the application is not referring to the device, it is referring to which platform are you creating an experiment, here we are talking about the `experiment.application`, when you go to create new experiment, you will see an option to select application- 
![image](https://user-images.githubusercontent.com/25848231/228371276-f6e1b65b-2fb8-421f-b4a1-36603a19e68e.png)

Reply:
Thanks @yashikakhurana I'll take a quick look